compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/lower/EnumClassLowering.kt

```diff
@@ -85,7 +89,22 @@ private class EnumClassLowering(val context: JvmBackendContext) : ClassLoweringP
                 annotations += enumEntry.annotations
             }
 
-        private fun buildValuesField(): IrField = irClass.addField {
+        private fun buildValuesHelperFunction(): IrFunction = irClass.addFunction {
+            name = Name.identifier(ImplementationBodyCodegen.ENUM_VALUES_FIELD_NAME.toLowerCase())
```

Comment:
Please spell out `"$values"` literally here, or use `toLowerCaseAsciiOnly` (see 77a9d14f93a8a0581f276bee3a1e211781791aee)

Reply:
Good to learn ASCII only variants! Spelled out instead, and made it a constant.